go/plek.go

```diff
@@ -7,6 +7,8 @@ import (
 	"strings"
 )
 
+const DevDomain = "dev.gov.uk"
```

Comment:
@alext: i've changed it to be a non-exported field, because it doesn't need to be exported.